tests/test_utilities.py

```diff
@@ -161,6 +161,16 @@ def test_noop(case, expected):
     assert _.noop(*case) == expected
 
 
+@parametrize('args,pos,expected', [
+    ([11, 22, 33, 44], 0, 11),
+    ([11, 22, 33, 44], -1, 44),
+    ([11, 22, 33, 44], -4, 11)
```

Comment:
Should add some test cases for `pos` that is higher/lower than what's given (e.g. 4 and -5) and other cases for default value, string pos, float pos, etc.